org.eclipse.xtext.tests/src/org/eclipse/xtext/resource/ResourceSetTest.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2012, 2020 itemis AG (http://www.itemis.eu) and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.xtext.resource;
+
+import java.util.HashMap;
+
+import org.eclipse.emf.ecore.resource.impl.ResourceSetImpl;
+
+public class ResourceSetTest extends AbstractResourceSetTest {
+	@Override
+	protected ResourceSetImpl createEmptyResourceSet() {
+		ResourceSetImpl rs = new ResourceSetImpl();
+		rs.setURIResourceMap(new HashMap<>());
```

Comment:
You might like to use `Collections.emptyMap()` here.

Reply:
want to keep semantics

Reply:
The resourceSet will later populate the map. `#emptyMap()` will fail on that attempt.